src/renderers/testing/stack/ReactTestRendererStack.js

```diff
@@ -50,17 +50,21 @@ function getRenderedHostOrTextFromComponent(component) {
   return component;
 }
 
+var UNSET = {};
+
 class ReactTestComponent {
   _currentElement: ReactElement;
   _renderedChildren: null | Object;
   _topLevelWrapper: null | ReactInstance;
   _hostContainerInfo: null | Object;
+  _nodeMock: Object;
 
   constructor(element: ReactElement) {
     this._currentElement = element;
     this._renderedChildren = null;
     this._topLevelWrapper = null;
     this._hostContainerInfo = null;
+    this._nodeMock = UNSET;
```

Comment:
Hmm, that's trickier. Probably not since nodes are supposed to be kept intact. I'm actually not sure the API is right now, maybe we should've just passed the `type` instead of the `element`. But what's done is done?